sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -0,0 +1,63 @@
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+import { createReadStream } from "fs";
+
+const readStream = createReadStream("./sample.jmx");
+dotenv.config({ path: "../../sample.env" });
+
+async function main() {
+  const Endpoint = process.env["LOADTESTSERVICE_ENDPOINT"] || "";
+  const DISPLAY_NAME = "some-load-test";
+  const SUBSCRIPTION_ID = process.env["SUBSCRIPTION_ID"] || "";
+  const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
```

Comment:
Let's use camel case for the variable names. `endpoint` and `client`. Also adding the client type is not needed, I'd recommend removing it to make the sample more concise.

```ts
const client = AzureLoadTesting(endpoint, new DefaultAzureCredential())
```

Reply:
Done.